<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductDialogWidget</class>
 <widget class="QDialog" name="AddProductDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление товара</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цена, р.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
